### DISCUSSION TITLE
website not updating (GitHub website used for upload and delete)

### DISCUSSION THREAD

**Discussion by yhcqw** [community/community#138807] [Actions] [answered]
### Select Topic Area

General

### Body

I dont know whether it is the right place to ask.
My website isn't updating. I use the GitHub website to upload and delete, not the command line. I deleted a file called sample.txt. It doesn't  not appear in my account anymore but when I open the link (something like https://xxx.github.io/xxx/sample.txt), it is still there. When I upload other html documents, the situation is the same. Files uploaded successfully, but the old website shows.
I updated Actions Permissions. It is now "Allow all actions and reusable workflows!" but still no changes.

**Comment by phuyalgaurav** [comment]
Make sure you've deployed the main branch.
>Sould look something like this
![image](https://github.com/user-attachments/assets/e43664f9-5b89-4053-8f15-2e3fafa41511)

**Reply by yhcqw** [reply]
Thank you for your answer.Yes, it is always in the main branch but not updating...

**Reply by yhcqw** [reply, marked as answer]
Issue solved now. I actually did nothing but just checked around ....Actions and Environment,and the issue just solved.
